javascript/javascript2/week1/currencyConverter.js

```diff
@@ -0,0 +1,82 @@
+// currency-converter-script.js
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Note that this line is optional since you have imported the script at the end of the `<body>` tag.

Reply:
Ok, Noted and Thank you very much!!

Reply:
I removed above line. Thank you!